get_image_predictions.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import pickle
+from model_prediction import *
+
```

Comment:
Sorry - let me walk that back a bit. Create a Driver (workbench.py) in the root, then have a "helpers" or similar for your extraneous helper code.